packages/react-components/react-button/package.json

```
@@ -39,6 +39,7 @@
     "@fluentui/react-tabster": "^9.5.3",
     "@fluentui/react-theme": "^9.1.5",
     "@fluentui/react-utilities": "^9.6.0",
+    "@fluentui/react-shared-contexts": "^9.2.0",
```

_nit:_ Keep alphabetical ordering

```suggestion
    "@fluentui/react-shared-contexts": "^9.2.0",
    "@fluentui/react-tabster": "^9.5.3",
    "@fluentui/react-theme": "^9.1.5",
    "@fluentui/react-utilities": "^9.6.0",
```